### PR TITLE
Jia/_/storybook-enhancement

### PR DESCRIPTION
**Pull request by jia-deriv**

### CODE REVIEW

File: lib/components/Link/link.stories.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ const meta = {
     title: "Components/Links/Link",
     component: Link,
     parameters: {
-        layout: "centered",
+        // layout: "centered",
```
Comment (prince-deriv) [inline lib/components/Link/link.stories.tsx:17]: Can we remove instead?

File: lib/components/Tag/custom-tag.stories.tsx [hunk 2]
```diff
@@ -52,9 +53,10 @@ const meta = {
             control: "boolean",
         },
         color: {
-            table: {
-                disable: true,
-            },
+            // table: {
+            //     disable: true,
+            // },
```
Comment (prince-deriv) [inline lib/components/Tag/custom-tag.stories.tsx:58]: let's remove this if not needed

**Review by prince-deriv — CHANGES_REQUESTED** [review]

File: lib/components/Link/link.stories.tsx (hunk 1 above)
Comment (jia-deriv) [reply lib/components/Link/link.stories.tsx]: Sure

File: lib/components/Tag/custom-tag.stories.tsx (hunk 2 above)
Comment (jia-deriv) [reply lib/components/Tag/custom-tag.stories.tsx]: Sure

**Review by prince-deriv — APPROVED** [review]

**Pull request merged**